utils/removeDiscordRole.js

```diff
@@ -0,0 +1,36 @@
+/*
+  This function handles all scenarios for role arguments - only "roleid" / only "rolename" / both.
+  Ensure that if only "roleid" is provided, pass "rolename" as "undefined", and vice versa.
+  When both "roleid" and "rolename" are available, pass both as arguments.
+  For implementation reference, see linkExternalAccount in controllers/external-accounts.js."
+ */
```

Comment:
- let's make this short and please follow jsdoc conventions.

Reply:
Rewritten it using jsdoc convention. Can you please check if it is correct?